llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -456,6 +479,19 @@ void saveModule(std::vector<std::unique_ptr<util::SimpleTable>> &OutTables,
   }
 }
 
+void saveDeviceLibModule(
+    std::vector<std::unique_ptr<util::SimpleTable>> &OutTables, int I,
+    LLVMContext &Context, const std::string &DeviceLibFileName) {
```

Comment:
It is better to create a local `LLVMContext` instead of passing it through functions.

Reply:
Hi, @maksimsab 
Done.
Thanks very much.